core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -138,26 +157,28 @@ private synchronized void reconcileKernelRegistration() {
         checkEnabled();
         Map<String, Object> previous = activeTriggers.remove(name);
 
-        try (Transaction tx = apocConfig.getSystemDb().beginTx()) {
+        withSystemDb(tx -> {
             tx.findNodes(SystemLabels.ApocTrigger,
-                    SystemPropertyKeys.database.name(), db.databaseName(),
-                    SystemPropertyKeys.name.name(), name)
+                            SystemPropertyKeys.database.name(), db.databaseName(),
+                            SystemPropertyKeys.name.name(), name)
                     .forEachRemaining(node -> node.delete());
-            tx.commit();
-        }
+            setLastUpdate(tx);
+            return null;
+        });
         updateCache();
         return previous;
     }
 
     public Map<String, Object> updatePaused(String name, boolean paused) {
         checkEnabled();
-        try (Transaction tx = apocConfig.getSystemDb().beginTx()) {
+        withSystemDb(tx -> {
             tx.findNodes(SystemLabels.ApocTrigger,
-                    SystemPropertyKeys.database.name(), db.databaseName(),
-                    SystemPropertyKeys.name.name(), name)
+                            SystemPropertyKeys.database.name(), db.databaseName(),
+                            SystemPropertyKeys.name.name(), name)
                     .forEachRemaining(node -> node.setProperty(SystemPropertyKeys.paused.name(), paused));
-            tx.commit();
-        }
+            setLastUpdate(tx);
```

Comment:
why not call `setLastUpdate` in `withSystemDb` directly?

Reply:
becuse we use `withSystemDb` also for reading ops like here:

https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/c1b34daf825933881ca91691dcb2752d3b914e20/core/src/main/java/apoc/trigger/TriggerHandler.java#L91-L105